site2/website/pulsar-manager-release-notes.md

```diff
@@ -1,5 +1,19 @@
 ## Apache Pulsar Manager
 
+### 0.3.0 &mdash; 2022-05-25 <a id="0.3.0"></a>
+* Fixed mysql schema field token error [PR-339](https://github.com/apache/pulsar-manager/pull/339)
+* Fixed topics List page, search Topics bug [PR-342](https://github.com/apache/pulsar-manager/pull/342)
+* Fixed pulsar manager's swagger dose not work. [PR-369](https://github.com/apache/pulsar-manager/pull/369)
+* Add offload threshold input [PR-375](https://github.com/apache/pulsar-manager/pull/375)
+* Add dashboard service for aggregated information collection [PR-326](https://github.com/apache/pulsar-manager/pull/326)
+* Multi bookie cluster Support [PR-388](https://github.com/apache/pulsar-manager/pull/388)
+* Fixed the bug that admin roles cannot be deleted. [PR-397](https://github.com/apache/pulsar-manager/pull/397)
```

Comment:
Use a period or not? please keep consistent

Reply:
fixed, thanks.